### PR TITLE
Adding KNN classifier

### PR DESCRIPTION
**Pull request by VishnuBhaarath**
The PR fix for issues #7 

Added KNN classifier to models.py the advantage of this is it takes almost zero time to train because it only stores the data of the training part. and faster than all the models mentioned in the models.py file , it is also a non parametric models with only parameter that needs to be mentioned is the number of neighbours, adding to this as KNN doesnt undergo training we can add new data to it which doesn't affect the accuracy of the model. It is also very easy to implement and interpret as there is only one hyperparameter which is the number of neighbours .

### CODE REVIEW

File: tab_automl/automl/models.py [hunk 1]
```diff
@@ -21,6 +22,7 @@
         "Decision Tree Classifier": DecisionTreeClassifier,
         "Light Gradient Boosting Classifier": LGBMClassifier,
         "Random Forest Classifier": RandomForestClassifier,
-        "XGBoost Classifier": XGBClassifier
+        "XGBoost Classifier": XGBClassifier,
+        "KNN classifier":KNeighborsClassifier
```
Comment (sagnik1511) [inline tab_automl/automl/models.py]: add a space after the colon

**Comment by sagnik1511** [comment]
@VishnuBhaarath, Please update a single at any instance on a single repo.

File: tab_automl/automl/models.py (hunk 1 above)
Comment (sagnik1511) [inline tab_automl/automl/models.py]: Make the `c` capital for the `classifier` in the dictionary key.

Comment (VishnuBhaarath) [reply tab_automl/automl/models.py]: > add a space after the colon

yeah will do it

Comment (VishnuBhaarath) [reply tab_automl/automl/models.py]: > Make the `c` capital for the `classifier` in the dictionary key.

yeah will do it

**Comment by VishnuBhaarath** [comment]
> @VishnuBhaarath, Please update a single at any instance on a single repo.

have added space as well as made c capital can you please elaborate on this comment, thanks

**Comment by sagnik1511** [comment]
@VishnuBhaarath , Please change the branch name as stated in `README.md`

**Comment by VishnuBhaarath** [comment]
> @VishnuBhaarath , Please change the branch name as stated in `README.md`

yeah sure will do it.

**Comment by VishnuBhaarath** [comment]
> > @VishnuBhaarath , Please change the branch name as stated in `README.md`
> 
> yeah sure will do it.

Have made a new pull request with my github username as a branch, kindly check it and can this PR be closed?

**Pull request merged**